wiki/Beatmap_Editor/Menu/en.md

```diff
@@ -1,441 +1,256 @@
+Beatmap Editor/Menu
+====================
+
 File
-====
-
-<img src="M_File.jpg" title="fig:Right|File menu" alt="Right|File menu" width="200" /> **Basic File commands.**
-
-<table>
-<thead>
-<tr class="header">
-<th><p>File menu</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>colspan=2| File commands</p></td>
-</tr>
-<tr class="even">
-<td><p>Clear All Notes<br />
-(Ctrl + N)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Open Difficulty...</p></td>
-</tr>
-<tr class="even">
-<td><p>Save<br />
-(Ctrl + S)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Create a new Difficulty...</p></td>
-</tr>
-<tr class="even">
-<td><p>colspan=2| Revert commands</p></td>
-</tr>
-<tr class="odd">
-<td><p>Revert to Saved<br />
-(Ctrl + L)</p></td>
-</tr>
-<tr class="even">
-<td><p>Revert to Saved (Full)<br />
-(Ctrl + Shift + L)</p></td>
-</tr>
-<tr class="odd">
-<td><p>colspan=2| Troubleshoot commands</p></td>
-</tr>
-<tr class="even">
-<td><p>Test Beatmap<br />
-(F5)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Open AiMod<br />
-(Ctrl + Shift + A)</p></td>
-</tr>
-<tr class="even">
-<td><p>colspan=2| Beatmap Packaging commands</p></td>
-</tr>
-<tr class="odd">
-<td><p>Upload Beatmap...<br />
-(Ctrl + Shift + U)</p></td>
-</tr>
-<tr class="even">
-<td><p>Export Package...</p></td>
-</tr>
-<tr class="odd">
-<td><p>Extract Map Package</p></td>
-</tr>
-<tr class="even">
-<td><p>Import from...</p></td>
-</tr>
-<tr class="odd">
-<td><p>colspan=2| Programmer's commands</p></td>
-</tr>
-<tr class="even">
-<td><p>Open Song Folder</p></td>
-</tr>
-<tr class="odd">
-<td><p>Open .osu in Notepad</p></td>
-</tr>
-<tr class="even">
-<td><p>Open .osb in Notepad</p></td>
-</tr>
-<tr class="odd">
-<td><p>colspan=2| Emergency Exit command.</p></td>
-</tr>
-<tr class="even">
-<td><p>Exit...<br />
-(Esc)</p></td>
-</tr>
-</tbody>
-</table>
+------
+
+![File menu](M_File.jpg "File menu")
+
+**Basic File commands.**
+
+### File commands
+
+| Name | Description |
+| ---- | ----------- |
+| Clear All Notes (Ctrl + N) | Purge the current difficulty's hitnotes. |
+| Open Difficulty... | **For Editing:** Quickly switch between difficulties. **For Reference:** (Mania Only) Overlay the selected difficulty over the (currently) opened one. |
+| Save (Ctrl + S) | Save current changes (both .osu and .osb) |
```

Comment:
change `Ctrl + S` to `` `Ctrl` + `S` ``
also change `.osu and .osb` to `` `.osu` and `.osb` ``